app/screens/contact/contact.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { 
+  View, Text
+ } from 'react-native';
+import style from './styles';
+
+ class ContactScreen extends Component {
+
+  render() {
+    return (
+      <View style={ style.mainContainer }>
+        
+      </View>
+    );
+  }
+
+ }
+
+ export default ContactScreen
```

Comment:
semicolon at end of statement

Reply:
Please follow coding standards from https://lftechnology.atlassian.net/secure/RapidBoard.jspa?rapidView=306&view=detail&selectedIssue=BWA-388&quickFilter=1018

Reply:
🆗